cortex/preflight_checker.py

```
@@ -0,0 +1,605 @@
+            self.check_software(software)
+        
+        # Calculate requirements
+        self.calculate_requirements(software)
+        
+        return self.report
+
+
+def format_report(report: PreflightReport, software: str) -> str:
+    """Format the preflight report for display"""
+    lines = []
+    lines.append(f"\n🔍 Simulation mode: No changes will be made\n")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)